apps/prairielearn/src/lib/course-request.sql

```diff
@@ -50,7 +50,10 @@ FROM
   LEFT JOIN select_course_request_jobs AS j ON j.id = r.id
 WHERE
   $show_all = 'true'
-  OR r.approved_status NOT IN ('approved', 'denied');
+  OR r.approved_status NOT IN ('approved', 'denied')
+ORDER BY
+  r.created_at DESC,
+  r.id DESC;
```

Comment:
Change in behavior: we now have consistent ordering of course requests, with the most recent ones first.